test/sql/catalog/test_extension_suggestion.test

```diff
@@ -1,12 +0,0 @@
-# name: test/sql/catalog/test_extension_suggestion.test
-# description: The error messages that suggests extension to be installed
-# group: [catalog]
-
-require skip_reload
-
-require no_extension_autoloading "EXPECTED: This tests what happens when extension is not there"
-
-statement error
-SELECT get_substrait("select 1");
-----
-Catalog Error: Scalar Function with name "get_substrait" is not in the catalog, a function by this name exists in the substrait extension, but it's of a different type, namely Table Function
```

Comment:
Maybe this we can keep, but with a different function / extension pair?

But that can be done independently.